Anlab.Mvc/Views/Lab/Confirmation.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using Humanizer
+@model AnlabMvc.Models.Order.OrderReviewModel
+
+@{
+    ViewData["Title"] = "Confirmation";
+}
+
+<h2>Confirmation</h2>
```

Comment:
@laholstege Don't need the h2 here because it is using the title. Feel free to remove it from the order confirmation page too.